.travis.yml

```diff
@@ -25,7 +25,16 @@ jobs:
       script:
         - echo "$DOCKER_PASSWORD" | docker login -u "ryanthemanuel" --password-stdin
         - travis_wait docker build -t cerner/terra-node -f ./docker/terra-node/Dockerfile ./docker/terra-node/
+        - travis_wait docker build -t cerner/terra-node-base:latest -f ./docker/terra-node-base/Dockerfile ./docker/terra-node-base/
+        - travis_wait docker build -t cerner/terra-node-ci:latest -f ./docker/terra-node-ci/Dockerfile ./docker/terra-node-ci/
+        - travis_wait docker build -t cerner/terra-node-zsh:latest -f ./docker/terra-node-zsh/Dockerfile ./docker/terra-node-zsh/
         - travis_wait docker push cerner/terra-node
+        - travis_wait docker push cerner/terra-node-base:latest
+        - travis_wait docker push cerner/terra-node-base:1
+        - travis_wait docker push cerner/terra-node-ci:latest
+        - travis_wait docker push cerner/terra-node-ci:1
+        - travis_wait docker push cerner/terra-node-zsh:latest
+        - travis_wait docker push cerner/terra-node-zsh:1
```

Comment:
I'm adding versions to these images to allow us to upgrade them more easily in the future.

Reply:
Also I'm adding new docker images to keep this passive. We could remove terra-node in the future.